internal/pkg/routes/route.go

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Mainflux
+// SPDX-License-Identifier: Apache-2.0
+
+package routes
+
+import (
+	"fmt"
+
+	mqtt "github.com/eclipse/paho.mqtt.golang"
+	"github.com/mainflux/mainflux/logger"
+	"github.com/nats-io/nats.go"
+)
+
+// Route - message route, tells which nats topic messages goes to which mqtt topic.
+// Later we can add direction and other combination like ( nats-nats).
+// Route is used in mfx and plain. Route is like base implementation and mfx and plain
+// are extended implemntation.
+type Route struct {
+	NatsTopic string
+	MqttTopic string
+	Subtopic  string
+	Logger    logger.Logger
+	Mqtt      mqtt.Client
+}
+
+type RouteIF interface {
```

Comment:
Follow Go interface naming [recommendation](https://golang.org/doc/effective_go.html#interface-names): `RouteHandler` or `RouteManager` or something similar, but please do not invent your own conventions like `IF` capitalized suffix.

Reply:
Something is wrong here - you have an interface and its implementation in the same file? What is this implementation and is it needed here?

Reply:
it is not uncommon to have interface and implementation in the same file.
check the comment here https://github.com/mainflux/export/pull/6#discussion_r365735708
RouteIF is for the interface 
I need Route struct to be public as well as interface so I cant use `route` for struct and `Route` for interface as we usually do